src/boot/efi/boot.c

```diff
@@ -115,8 +115,6 @@ static BOOLEAN line_edit(
         len = StrLen(line);
         print = AllocatePool((x_max+1) * sizeof(CHAR16));
 
-        uefi_call_wrapper(ST->ConOut->EnableCursor, 2, ST->ConOut, TRUE);
```

Comment:
why not show both? there might be outputs where color is not available, i'd rather be safe than sorry

Reply:
That looks awful and should be unnecessary: 
![image](https://user-images.githubusercontent.com/7892635/127685078-694bc353-acf8-4c2d-905f-70b49fc88089.png)

Reply:
i am bit concerned about relying only on color here. at least show some caret-like glyph?

Reply:
The glyph we're using is a spacebar. And reversing the color has always worked in the boot menu, so this be fine here, no?

Reply:
space is not really a glyph. 

i do accept your point about the boot menu selection bar though. still feels dangerous